generator/src/main/java/com/scottlogic/deg/generator/generation/string/iterators/FiniteStringAutomatonIterator.java

```diff
@@ -53,34 +56,30 @@ public boolean hasNext() {
         }
 
         while (true) {
-            if (nextValue != null) {
-                return true;
-            } if (stateTree.isEmpty()) {
-                return false;
+            if (nextValue != null) return true;
```

Comment:
{}

though it may be worth having a discussion on whether the current styling decision is the right one, it's not been challenged for a while